<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelaOpcaoCor</class>
 <widget class="QDialog" name="janelaOpcaoCor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mude as cores</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imagens/logo3.png</normaloff>../imagens/logo3.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>271</width>
     <height>211</height>
    </size>
   </property>
   <property name="title">
    <string>Cores</string>
   </property>
   <widget class="QDialogButtonBox" name="buttonBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Maior Elevação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="caixaCorMaiorElevacao">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Menor Elevação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="caixaCorMenorElevacao">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Rede de drenagem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="caixaCorRedeDeDrenagem">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Alagamento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="caixaCorAlagamento">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Barragem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="caixaCorBarragem">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="corPadraoCaixa">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>178</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cores Padrão</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>janelaOpcaoCor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>janelaOpcaoCor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
